gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/sharedpolicygroup/SharedPolicyGroup.java

```diff
@@ -74,7 +77,7 @@ public List<Plugin> getPlugins() {
     public enum Phase {
```

Comment:
hehe i got the same one at fix i don't know where. i'm thinking that in a next PR we could use the type that comes from plugin. wdyt ?🤔